llvm/lib/Target/RISCV/RISCVTargetTransformInfo.cpp

```diff
@@ -863,6 +863,21 @@ RISCVTTIImpl::getIntrinsicInstrCost(const IntrinsicCostAttributes &ICA,
     }
     break;
   }
+  case Intrinsic::get_active_lane_mask: {
+    if (ST->hasVInstructions()) {
+      Type *ExpRetTy = VectorType::get(
+          ICA.getArgTypes()[0], cast<VectorType>(RetTy)->getElementCount());
+      auto LT = getTypeLegalizationCost(ExpRetTy);
+
+      // vid.v   v8  // considered hoisted
```

Comment:
Don't we need to count it for code size?

Reply:
This poses a dilemma as both statements hold true:
1) the `vid.v` would be hoisted.
2) BasicImpl doesn't count it for all CostKind.

so I am leaning toward to consistency with the base implementation.

Reply:
ok